assets/merging-fee.md

```diff
@@ -0,0 +1,63 @@
+# Merge Fee -- FAQ
+
+**Someone is asking for cypto payment for merging my Pull Request.  Is this a scam?**
```

Comment:
don't think this part is needed

Reply:
Well, this is the question received most often so far, I think it makes our life easier.